.eslintrc.js

```diff
@@ -27,9 +27,9 @@ module.exports = {
     'comma-spacing': ['warn'],
     'curly': ['warn', 'all'],
     'eqeqeq': ['warn'],
-    'indent': ['warn', 2],
-    'key-spacing': ['warn'],
-    'keyword-spacing': ['warn'],
+    'indent': ['error', 2],
```

Comment:
maybe change also another automatically fixable rules to error ?

Reply:
Done